testing/test_predict.py

```diff
@@ -0,0 +1,35 @@
+import librispect as lspct
+from librispect.features import spectrogram, predict
+import glob
+import librosa
+import numpy as np
+import pytest
+
+SPECT_HEIGHT = 2048
+STFT_HEIGHT = 2048
+WINDOW_SIZE = 16
+BATCH_SIZE = 32
+STEP_SIZE = 4
+TERMS = 8
+PREDICT_TERMS = 8
+
+def test_spect_predict_maker():
+    '''integration testing of prediction generation'''
+    hparams = lspct.features.spectrogram.HPARAMS
+    test_spect_predict_maker = predict.spect_predict_maker(hparams, terms=TERMS, predict_terms=PREDICT_TERMS)
+    path_list = glob.glob((lspct.paths.WAV_DIR / '*.wav').as_posix())[0] 
```

Comment:
this means you take the wav list and take the first element and store it as pathlist

then later when you iterate through path_list it will iterate through the characters of the path (and crash on the first one saying '/' is a directory)